.github/workflows/test-bridge-pr.yml

```diff
@@ -0,0 +1,95 @@
+name: "[Test] Bridge"
+
+on: pull_request
+
+concurrency:
+  group: ci-bridge-${{ github.ref }}
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  is-affected:
+    outputs:
+      is-affected: ${{ steps.affected.outputs.is-affected }}
+      paths: ${{ steps.affected.outputs.paths }}
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
+          fetch-depth: 0
+      - name: checkout branch (PR)
+        if: ${{ github.event_name == 'pull_request' }}
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh pr checkout ${{ github.event.pull_request.number }}
+      - name: checkout branch (not PR)
+        if: ${{ github.event_name != 'pull_request' }}
+        run: |
+          git checkout $GITHUB_SHA
+      - uses: ./tools/actions/live-common-affected
+        id: affected
+        with:
+          ref: ${{ github.event_name == 'pull_request' && format('origin/{0}', github.event.pull_request.base.ref) || 'HEAD^' }}
+
+  bridge-tests:
+    name: "Bridge Tests"
+    needs: is-affected
+    if: ${{ needs.is-affected.outputs.is-affected == 'true' }}
+    env:
+      NODE_OPTIONS: "--max-old-space-size=7168"
+      FORCE_COLOR: 3
+      CI_OS: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
```

Comment:
Do we want to run bridge test on all OS? Can we have differences or is Linux enough?

Reply:
This is something I thought about aswell, Linux should be enough but I decided to let all platforms for now as a safety net, but you make me question myself now :sweat_smile:

I'm doing this in another PR

Reply:
I ask because we will remove macOS and Windows for e2e tests 😀 https://github.com/LedgerHQ/ledger-live/pull/6098